### ISSUE TITLE
invalid vanilla enchantment console error

### ISSUE THREAD

**Issue by xlsalvolx** [GC-spigot/AdvancedEnchantments#4329]
### Describe the bug

This is the custom enchant causing the issue:

```
soulbound:
  display: '%group-color%Soulbound'
  description: You may keep your item on death
  applies-to: every weapon and armor
  type: DEATH;PASSIVE_DEATH
  group: SOULBOUND
  applies:
  - PLAYER_HEAD
  - DRAGON_HEAD
  - SKELETON_SKULL
  - WITHER_SKELETON_SKULL
  - ZOMBIE_HEAD
  - CREEPER_HEAD
  - ALL_TOOLS
  - ALL_SWORD
  - ALL_AXE
  - ALL_ARMOR
  - BOW
  - CROSSBOW
  - TRIDENT
  - PRISMARINE_CRYSTALS
  - NETHER_STAR
  - BONE
  - SHIELD
  - BLAZE_ROD
  - STICK
  - ALLIUM
  - TOTEM_OF_UNDYING
  - SPYGLASS
  - FIRE_CHARGE
  - GLOBE_BANNER_PATTERN
  - NETHER_STAR
  levels:
    '1':
      chance: 100
      effects:
      - REMOVE_ENCHANT:soulbound:1
      - KEEP_ON_DEATH
    '2':
      chance: 100
      effects:
      - REMOVE_ENCHANT:soulbound:2
      - ADD_ENCHANT:soulbound:1
      - KEEP_ON_DEATH
    '3':
      chance: 100
      effects:
      - REMOVE_ENCHANT:soulbound:3
      - ADD_ENCHANT:soulbound:2
      - KEEP_ON_DEATH
```

And this is the error appearing in my console over and over every time ADD_ENCHANT and REMOVE_ENCHANT triggers:

`[17:35:29 WARN]: [AdvancedEnchantments] Invalid vanilla enchantment: soulbound. Enchantment names can be found here: https://hub.spigotmc.org/javadocs/spigot/org/bukkit/enchantments/Enchantment.html`

This is happening only if ADD_ENCHANT or REMOVE_ENCHANT try to remove or add the enchant

### How to reproduce

1. Load the enchant
2. Enchant a sword with it and keep it in your inventory
3. Kill yourself
4. Check console for the error

### "/ae plinfo" link

https://paste.md-5.net/ayipesogir

**Comment by GC-spigot** [comment]
Solved!

**Comment by xlsalvolx** [comment]
> Solved!

Not solved, still happening as of 9.8.8

```
[13:07:54 INFO]: xlsalvolx issued server command: /ae enchant soulbound 3
[13:07:59 INFO]: xlsalvolx issued server command: /kill xlsalvolx
[13:07:59 WARN]: [AdvancedEnchantments] Invalid vanilla enchantment: soulbound. Enchantment names can be found here: https://hub.spigotmc.org/javadocs/spigot/org/bukkit/enchantments/Enchantment.html
[13:07:59 INFO]: xlsalvolx was killed
[13:07:59 INFO]: [xlsalvolx: Killed xlsalvolx]
> ae plinfo
[13:10:16 INFO]: Creating info link, please wait.
[13:10:17 INFO]: https://paste.md-5.net/evasilojen
```
